python/mleap/pyspark/feature/math_binary.py

```diff
@@ -102,7 +105,8 @@ def setParams(self, inputA=None, inputB=None, outputCol=None):
         """
         Sets params for this MathBinary.
         """
-        kwargs = self._input_kwargs
+        # For the correct behavior of MathBinary, params that are None must be unset
```

Comment:
What happens if they are not unset?

Reply:
The spark hook for the transformer does a check for isSet(col1), which passes even when the col is None causing the transform to fail